src/isa.h

```diff
@@ -0,0 +1,272 @@
+/*
+  Copyright (c) 2025, Intel Corporation
+
+  SPDX-License-Identifier: BSD-3-Clause
+*/
+
+#ifndef ISA_H
```

Comment:
What about `#pragma once`?

Reply:
It is not part of the C or C++ standard.